changehc/delphi_changehc/sensor.py

```diff
@@ -46,12 +46,12 @@ def backfill(
             k=Config.MAX_BACKFILL_WINDOW,
             min_visits_to_fill=Config.MIN_CUM_VISITS):
         """
-        Adjust for backfill (retroactively added observations) by using a
-         variable length smoother, which starts from the RHS and moves
-         leftwards (backwards through time). We cumulatively sum the total
-         visits (denominator), until we have observed some minimum number of
-         counts, then calculate the sum over that bin. We restrict the
-         bin size so to avoid inluding long-past values.
+        Adjust for backfills (retroactively added observations) by using a variable length smoother.
```

Comment:
Is pydocstyle forcing you to use a plural here? We use backfill to mean the phenomenon of retroactively added observations, rather than referring to the observations themselves.

Reply:
it didn't like the fact that the summary had the exact same word as the method. I reordered in a way to make it pass, let me know if it reads ok.

Reply:
actually i think it's more confusing now since it's the wrong definition, one moment